src/flows/recommendation_api/corpus_candidate_sets/collections_by_recency.py

```diff
@@ -15,15 +15,30 @@
 RECENT_COLLECTIONS_CANDIDATE_SET_ID = "92af3dae-25c9-46c3-bf05-18082aacc7e1"
 
 EXPORT_COLLECTIONS_CANDIDATE_SET_SQL = """
-SELECT 
+WITH recent_collections AS (
+  SELECT
+    IFNULL(s.SCHEDULED_CORPUS_ITEM_SCHEDULED_AT, a.REVIEWED_CORPUS_ITEM_CREATED_AT) as recency,
+    s.SCHEDULED_SURFACE_IANA_TIMEZONE,
+    a.*
+  FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" a
+  LEFT JOIN "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS" s ON (
+    s.approved_corpus_item_external_id = a.approved_corpus_item_external_id
+    AND s.SCHEDULED_CORPUS_ITEM_SCHEDULED_AT < current_timestamp()
+    AND s.SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
```

Comment:
I am curious if the collections that are not scheduled will be associated with any scheduled surface.  I am also curious how collections get into the prospecting tool or the `approved_corpus_items` model at all.  They are not part of any prospect source, so I am worried they will not have an associated surface (which may be fine).

Reply:
Technically curators could submit Collections manually and recommend them, without being scheduled on New Tab. The last time that happened was in Dec 2021 for a 'most read articles' collection.